regulation-worker/internal/delete/batch/batch.go

```diff
@@ -347,35 +347,27 @@ func (b *Batch) upload(ctx context.Context, uploadFileAbsPath, actualFileName, a
 
 func (b *Batch) createPatternFile(userAttributes []model.UserAttribute) (string, error) {
 	pkgLogger.Debug("creating a file with pattern to be searched & deleted.")
-	var n int
 
-	for i := 0; i < len(userAttributes); i++ {
-		n += len(userAttributes[i].UserID) + 4
-		if userAttributes[i].Email != nil {
-			n += len(*userAttributes[i].Email) + 4
-		}
-		if userAttributes[i].Phone != nil {
-			n += len(*userAttributes[i].Phone) + 4
-		}
-	}
-
-	searchObject := make([]byte, 0, n)
+	searchObject := make([]byte, 0)
 
 	for _, users := range userAttributes {
 		searchObject = append(searchObject, "/"...)
+		searchObject = append(searchObject, "\"userId\": *\""...)
```

Comment:
prefer \s for space, https://www.rexegg.com/regex-quickstart.html it can much new-lines, tabs etc
```suggestion
		searchObject = append(searchObject, "\"userId\":\s*\""...)
```